support-metrics-client/src/main/java/io/confluent/support/metrics/SupportedKafka.java

```diff
@@ -18,7 +18,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.Locale;
```

Comment:
Why do we need `java.util.Locale`?

Reply:
`System.getProperty("os.name").toLowerCase(Locale.ROOT)`

Reply:
Makes sense, thanks.